<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollBar::sub-line:vertical {
background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
 stop: 0  rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130),  stop:1 rgb(32, 47, 130));
height: 0 px;
subcontrol-position: top;
subcontrol-origin: margin;
};
</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>517</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214)</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(214, 214, 214)</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="startAll"/>
   <addaction name="stopAll"/>
   <addaction name="edit"/>
   <addaction name="settings"/>
   <addaction name="timer"/>
  </widget>
  <action name="startAll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menu/images/start all.png</normaloff>:/menu/images/start all.png</iconset>
   </property>
   <property name="text">
    <string>Запустить всё</string>
   </property>
  </action>
  <action name="stopAll">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menu/images/stop all.png</normaloff>:/menu/images/stop all.png</iconset>
   </property>
   <property name="text">
    <string>Остановить всё</string>
   </property>
  </action>
  <action name="settings">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menu/images/settings.png</normaloff>:/menu/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Настройка</string>
   </property>
   <property name="toolTip">
    <string>Настройка</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="edit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menu/images/edit.png</normaloff>:/menu/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактирование</string>
   </property>
   <property name="toolTip">
    <string>Редактирование</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="timer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menu/images/timer.png</normaloff>:/menu/images/timer.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>Таймер</string>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
